tests/utilities_t/test_utils.py

```diff
@@ -106,6 +107,97 @@ def test_csv_parser_slurm(self):
         )
 
 
+class TestDeepUpdate(TestCase):
+    def test_deep_update_simple_dict(self):
+        original_mapping = {"test": 123, "NoChange": 789}
+        mapping_update = {"test": 456}
+        updated_mapping = {"test": 456, "NoChange": 789}
+
+        self.assertDictEqual(
+            updated_mapping, deep_update(original_mapping, mapping_update)
+        )
+
+    def test_deep_update_nested_dict(self):
+        original_mapping = {
+            "test": {"test": 123},
+            "NoChange1": 123,
+            "NoChange2": {"Test": 789},
+        }
+        mapping_update = {"test": {"test": 123}}
+        updated_mapping = {
```

Comment:
Does this test actually update anything? Looks identical to me. 😓

Reply:
Hmm, that was bit different intended. Sorry. :-(

Reply:
Fixed